src/components/AppSidebar/AppSidebar.vue

```diff
@@ -387,20 +426,60 @@ export default {
 		},
 	},
 
+	updated() {
+		this.$nextTick(this.forceUpdateClamps)
```

Comment:
Temporary hack, for some reason the docs doesn't trigger the open/close properly.
On real life example it seems to works fine :thinking: 

![Capture d’écran_2021-03-25_16-39-46](https://user-images.githubusercontent.com/14975046/112500564-c8de1b80-8d88-11eb-90ad-f67652823371.png)